test/e2e/storage/csi_mock_volume.go

```diff
@@ -1165,6 +1209,97 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			})
 		}
 	})
+
+	// These tests *only* work on a cluster which has the CSIVolumeFSGroupPolicy feature enabled.
+	ginkgo.Context("CSI FSGroupPolicy", func() {
+		tests := []struct {
+			name          string
+			fsGroupPolicy storagev1.FSGroupPolicy
+		}{
+			{
+				name:          "should modify fsGroup if fsGroupPolicy=File",
+				fsGroupPolicy: storagev1.FileFSGroupPolicy,
+			},
+			{
+				name:          "should not modify fsGroup if fsGroupPolicy=None",
+				fsGroupPolicy: storagev1.NoneFSGroupPolicy,
+			},
+		}
+		for _, t := range tests {
+			test := t
+			ginkgo.It(test.name, func() {
+				if !utilfeature.DefaultFeatureGate.Enabled(FeatureCSIVolumeFSGroupPolicy) {
+					e2eskipper.Skipf("FSGroupPolicy is not enabled -- skipping")
+				}
```

Comment:
we can't validate feature gate in e2e. Since we are moving the feature to beta, we don't need to.

Reply:
This check has been removed.